.github/workflows/jobs.yml

```diff
@@ -284,9 +300,13 @@ jobs:
       - name: Run job
         run: |
           mkdir -p build && cd build
-          cmake -GNinja -DICB=ON -DEXAMPLES=ON -DMPI=ON ..
+          if [[ ${{ matrix.int64 }} == ON ]]; then
+            _blas_lib="-DBLAS_LIBRARIES=openblas_64"
+          fi
+          cmake -GNinja -DICB=ON -DEXAMPLES=ON -DMPI=${{ matrix.mpi }} -DINTERFACE64=${{ matrix.int64 }} ${_blas_lib} ..
           cmake --build . -v
       - name: Run tests
+        continue-on-error: ${{ matrix.msystem == 'CLANG64' }}
```

Comment:
Are you sure this fails if the job fails? (red CI) If not, please remove this, thanks

Reply:
Flang is not ready yet, This will ignore the failure of tests with it.

Reply:
As this project is maintained-only (no development), I prefer to have in CI only what works. There is nobody to fix bug or maintain this project.

Can you just run in CI what works: if `flang` (whatever version) could be used that's good new, otherwise there is no point. When `flang` is ready, we'll add it in the CI. @sylvestre: what do you say?

Reply:
@MehdiChinoune: you said flang is used on CLANG64, and, this test fails, right?
If so no point to me to add such a job in CI (at least for now - until `flang` is ready)

Reply:
You were the one asking for a CI job.
The original PR was to add the required flag to build arpack-64 with flang.

Reply:
> asking for a CI job.

Yes, thanks for that. In the first place, I was thinking of https://apt.llvm.org/

Reply:
> The original PR was to add the required flag to build arpack-64 with flang.

Feel free to improve CI if you think this can be covered